crates/database/src/orders.rs

```diff
@@ -1253,6 +1267,20 @@ mod tests {
         let mut db = db.begin().await.unwrap();
         crate::clear_DANGER_(&mut db).await.unwrap();
 
+        let metadata: serde_json::Value = serde_json::from_str(
+            r#"{ "interactions": [ {
+            "target": "0102030405060708091011121314151617181920",
+            "value": 2.1,
```

Comment:
FYI `value` will always contain integers and not floats. Not that it matters for the tests, though.

Reply:
I've changed this to integer to be consistent with the data model, as it will not use floats.